packages/esm-tb-app/translations/en.json

```diff
@@ -39,6 +39,8 @@
   "tptAdherence": "Adherence",
   "nextAppointmentDate": "Next Appointment Date",
   "previousCases":  "Previous TPT Cases",
-  "previousTptCases": "Recent TPT Cases"
+  "previousTptCases": "Recent TPT Cases",
+  "recentTuberculosis": "Recent Tuberculosis",
+  "previousCases": "Previous Cases"
```

Comment:
this fixes the TB summary as it has the 'Previous TPT Cases' header instead of 'Previous Cases'